config/application.yml.default

```diff
@@ -289,6 +289,7 @@ usps_ipp_username: ''
 usps_mock_fallback: true
 get_usps_proofing_results_job_cron: '0/10 * * * *'
 get_usps_proofing_results_job_reprocess_delay_minutes: 5
+get_usps_proofing_results_job_request_delay_seconds: 1
```

Comment:
Wondering if we'd want to be able to configure this for sub-second delays. Would milliseconds be a better unit of time?

Reply:
Yeah good idea. Changes it to ms 👍